src/ui/GroupsView.js

```diff
@@ -1,4 +1,6 @@
+import CenterCard from './CenterCard';
```

Comment:
ditto

Reply:
`GroupsView` isn't ready for Flow yet. It needs extra work to make it pass Flow.